ui/shared/components/graph/VariantTagTypeBar.jsx

```diff
@@ -5,22 +5,32 @@ import HorizontalStackedBar from '../graph/HorizontalStackedBar'
 import { EXCLUDED_TAG_NAME, REVIEW_TAG_NAME, NOTE_TAG_NAME } from '../../utils/constants'
 
 
-export const getVariantTagTypeCount = (vtt, familyGuids) => (
-  familyGuids ? familyGuids.reduce((count, familyGuid) => count + (vtt.numTagsPerFamily[familyGuid] || 0), 0) : vtt.numTags
-)
+export const getVariantTagTypeCount = (vtt, familyGuids, variantGuid, variantsToDisplay) => {
+  if (familyGuids) {
+    return familyGuids.reduce((count, familyGuid) => count + (vtt.numTagsPerFamily[familyGuid] || 0), 0)
+  }
+  else if (variantGuid) {
```

Comment:
From the ticket: 
```
variant-specific pages... should show the breakdown across only the tags in that specific family
```
What this code will do is show you the breakdown of tags for that single variant. However, this should be the breakdown of all the tags for the family. So what you want to do here is get the `familyGuids` from the variant and use those